jobs/config.json

```diff
@@ -8519,6 +8519,27 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7-upgrade-cluster-init": {
+    "args": [
+      "--cluster=",
+      "--deployment=kubernetes-anywhere",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=ci/latest-1.6",
```

Comment:
This means that the v1.6 version of the e2e.test binary will be used.
Should we/do we upgrade the e2e.test binary to v1.7 as well?
(I think we definitely should)

Reply:
Sure - how can we do that? Should I switch this arg to `ci/latest-1.7` or create a new config block?